src/pages/policy/PolicyRightSidebar.jsx

```diff
@@ -198,8 +199,56 @@ export default function PolicyRightSidebar(props) {
   const reformPolicyId = searchParams.get("reform");
   const baselinePolicyId = searchParams.get("baseline");
   const focus = searchParams.get("focus") || "";
+  const stateAbbreviation = focus.split(".")[2];
```

Comment:
This is a nice way of doing it. I guess it might not capture everything (e.g. if someone builds a reform including parameter changes at the state and federal level, and they're currently looking at a federal parameter) but this solves the immediate and most important use-case for sure.